docs/build/index.md

```diff
@@ -0,0 +1,2 @@
+# Operation Index
```

Comment:
I think something went wrong building the documentation.

Reply:
How now?

Reply:
The other docs files look good. _This_ one can be removed. And you should also re-build the schema:
```bash
python schema/build/generate_schema.py
```

Reply:
OK, got it